cypher/cypher-docs/src/docs/dev/syntax/naming.asciidoc

```diff
@@ -5,22 +5,30 @@ We describe here rules and recommendations for the naming of node labels, relati
 
```

Comment:
I know it's something you haven't changed but still I would remove the 'we' from the page and make it descriptive instead of personal. Same for the last sentence (our recommendations).

Reply:
Something like:
"This section describes the set of rules and recommendations to follow when naming node labels, relationship types, property names, and variables."

Reply:
Agree!